src/components/molecules/contactForm/ContactForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRef, useState } from "react"
+import { Box, Button } from "@mui/material"
+import { TextInput } from "../../atoms/formInput/TextInput"
+import emailjs from '@emailjs/browser';
+import { Input, inputs } from "../../../common/constants/enums";
+import { inputToFormProps } from "../../../common/constants/constants";
+
+const defaultFieldState:{[key in Input]: string} = inputs.reduce((acc, i) => {
+  acc[i] = ""
+  return acc
+}, {} as {[key in Input]: string})
+
+
+function ContactForm() {
+  const [fieldStates, setFieldStates] = useState(defaultFieldState)
+  const [formSubmitted, setFormSubmitted] = useState(false)
+
+  
+  function updateFieldState(field: Input, value: string) {
+    setFieldStates((currentFieldStates) => ({
+      ...currentFieldStates,
+      [field]: value,
+    }));
+  }
+
+  const form = useRef();
+
+  const sendEmail = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const currentForm = form.current
+
+    // TODO: enValid package
```

Comment:
💭 We probably don't need a package for this. All it's reall doing is accessing the env variables and making them available in an exported object.

Reply:
Yeah, it's why I didn't worry about it here. We can add it if we want later, but not necessary now.